openmdao/solvers/nonlinear/newton.py

```diff
@@ -190,6 +192,25 @@ def _iter_execute(self):
         # Disable local fd
         approx_status = system._owns_approx_jac
         system._owns_approx_jac = False
+        
+        # This cycles through the solvers of the subsystems once at the start of the problem
+        if self.options['initial_sub_solve'] == True:
+
+            for isub, subsys in enumerate(system._subsystems_allprocs):
+                system._transfer('nonlinear', 'fwd', isub)
+
+                if subsys in system._subsystems_myproc:
+                    subsys._solve_nonlinear()
+
+            system._apply_nonlinear()
+            print("Did initial sub-solve")
+            self.options['initial_sub_solve'] = False
+        
+            if do_subsolve == True:
+                print("WARNING: Both solve_subsystems and initial_sub_solve are set to True.",
+                "Setting solve_subsystems to False and carrying out one initial sub-solve only.")
+                self.options['solve_subsystems'] = False
+                do_subsolve = False
```

Comment:
Also, are there cases where we might want to run a few iterations of sub solves the first time we execute, and then run with no subsolves thereafter? I think this can be integrated so that all three options work together.